console/console-init/ui/src/modules/project/components/IoTConfiguration/IoTConfiguration.tsx

```diff
@@ -80,27 +80,32 @@ const IoTConfiguration: React.FC<IIoTConfigurationProps> = ({
               <FormGroup
                 label="Namespace"
                 isRequired={true}
-                fieldId="name-space"
+                fieldId="iot-config-namespace-selecttoggle"
               >
                 <SelectWithToggle
                   selectOptions={namespaceOptions}
                   onSelectItem={onNameSpaceSelect}
                   selections={namespace}
                   ariaLabel={"iot namespace"}
-                  id="iot-config-select-namespace"
+                  id="iot-config-name-space-selecttoggle"
```

Comment:
I think we should remove toggle from the last and make id as `iot-config-namespace-select`